events/wc_realm_events.txt

```diff
@@ -545,4 +545,158 @@ character_event = {
 		# Clear flags and other stuff
 		clear_settler_flag_effect = yes
 	}
+}
+
+### Ymiron Awakened ###
+character_event = {
+	id = WCRLM.799
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	immediate = {
+		# Saves ROOT's liege for future use
+		if = {
+			limit = { independent = no }
+			liege = { save_event_target_as = target_liege }
+		}
+		
+		# Creates Ymiron
+		create_character = {
+			name=Ymiron
+			dynasty=70000
+			dna="anolmcpmehb"
+			culture=vrykul religion=ROOT
+			attributes = { martial=6 diplomacy=6 stewardship=4 intrigue=7 learning=7 }
+			trait=skilled_tactician
+			trait=class_warrior_7
+			trait=wroth trait=proud trait=ambitious trait=arbitrary trait=paranoid
+			trait=creature_vrykul
+			random_traits = yes
+			age = 300
+		}
+		new_character = {
+			save_event_target_as = target_ymiron
+		}
+		event_target:target_ymiron = {
+			# Gives bloodline
+			create_bloodline = {
+				type = ymiron
+			}
+			
+			# Angerboda becomes your spouse
+			c_100001 = {
+				if = {
+					limit = {
+						OR = {
+							character = ROOT
+							AND = {
+								is_married = no
+								is_betrothed = no
+							}
+						}
+					}
+					if = {
+						limit = { is_married = yes }
+						any_spouse = { remove_spouse = PREV }
+					}
+					if = {
+						limit = { is_betrothed = yes }
+						betrothed = { break_betrothal = PREV }
+					}
+					add_spouse = event_target:target_ymiron
+					add_lover = event_target:target_ymiron
+				}
+			}
+			
+			# Ymiron gets ROOT titles, k_howling_fjord or claim on k_howling_fjord
+			event_target:target_ymiron = { ROOT = { this_abdicate_transfer_wealth_to_play_prev_effect = yes } }
```

Comment:
Don't transfer wealth, don't transfer artifacts.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768428-eaa27700-2f26-11ea-8b27-1af8122a92b3.png)
![image](https://user-images.githubusercontent.com/46576860/71768437-0279fb00-2f27-11ea-80a0-67505cb5f134.png)
![image](https://user-images.githubusercontent.com/46576860/71768451-09087280-2f27-11ea-8045-e48adcb8e9d1.png)

</details>

Reply:
Also don't clear claims

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/71768928-1aa04900-2f2c-11ea-84d6-17167a76d183.png)

</details>

Reply:
Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/f8fe146924d610f47a4280d438eb7ead907d5708 and https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/fb8761cd07dfb839fbf2ac1c71a4667be23e6134.